src/Main.java

```diff
@@ -1,5 +1,124 @@
 public class Main {
     public static void main(String[] args) {
 
+        //Задача 1
+
+        byte b = 5;
+        short s = 5000;
+        int i = 50000;
+        long l = 5000000L;
+        float f = 5.55f;
+        double d = 5.55555555;
+        System.out.println("\n" + "Задача 1\n");
+        System.out.println("Значение переменной b с типом byte равно " + b);
+        System.out.println("Значение переменной s с типом short равно " + s);
+        System.out.println("Значение переменной i с типом int равно " + i);
+        System.out.println("Значение переменной l с типом long равно " + l);
+        System.out.println("Значение переменной f с типом float равно " + f);
+        System.out.println("Значение переменной d с типом double равно " + d);
+
+        //Задача 2
+
+        float firstValue = 27.12f;
+        long secondValue = 987678965549L;
+        float thirdValue = 2.786f;
+        short fourthValue = 569;
+        short fifthValue = -159;
+        short sixthValue = 27897;
+        byte seventhValue = 67;
+        System.out.println("\n" + "Задача 2\n");
+        System.out.println("float firstValue = 27.12f;\n" +
+                        "long secondValue = 987678965549L;\n" +
+                        "float thirdValue = 2.786f;\n" +
+                        "short fourthValue = 569;\n" +
+                        "short fifthValue = -159;\n" +
+                        "short sixthValue = 27897;\n" +
+                        "byte seventhValue = 67;");
+
+        //Задача 3
+
+        byte lpStudents = 23;
+        byte asStudents = 27;
+        byte eaStudents = 30;
+        short totalSheets = 480;
+        short totalStudents = (short) (lpStudents + eaStudents + asStudents);
+        short sheetsPerStudent = (short) (totalSheets / totalStudents);
+        System.out.println("\n" + "Задача 3\n");
+        System.out.println("На каждого ученика рассчитано " + sheetsPerStudent + " листов бумаги");
+
+        //Задача 4
+
+        byte bottlesPerMin = 16 / 2;
+        short bottlesPer20Min = (short) (bottlesPerMin * 20);
+        short bottlesPerDay = (short) (bottlesPerMin * 1440);
+        int bottlesPer3Days = bottlesPerMin * 4320;
+        int bottlesPerMonth = bottlesPerMin * 44640;
```

Comment:
снова счёт в уме, а надо было всего лишь использовать правильно переменные:
```suggestion
        int bottlesPerMonth = bottlesPer3Days * 10;
```